tests/ui/components/results/verticalresultscomponent.js

```diff
@@ -48,6 +61,29 @@ describe('vertical results component', () => {
     };
   });
 
+  it('the query propery is based on global storage when the component is created', () => {
```

Comment:
query property?

Reply:
The component has a query property which it gets from global storage. If we don't think the test is that useful I can remove it

Reply:
I'll update this test to the hopefully more useful: 'getVerticalUrl encodes the global storage query when the component is created'